test/secrets_test.rb

```diff
@@ -139,6 +139,14 @@ def response_body(body)
       end
     end
 
+    describe 'timeouts' do
+      it 'times out with a get request' do
```

Comment:
I usually try to make tests describe a human readable expectation, something like "raises a helpful error message when connection fails"